source/retryable-writes/retryable-writes.rst

```diff
@@ -197,6 +197,18 @@ response.
 .. _findAndModify: https://www.mongodb.com/docs/manual/reference/command/findAndModify/
 .. _aggregate: https://www.mongodb.com/docs/manual/reference/command/aggregate/
 
+Retryable Writes Withing Transactions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In MongoDB 4.0 the only supported retryable write commands within a transaction
+are ``commitTransaction`` and ``abortTransaction``. Therefore drivers MUST NOT
+retry write commands within transactions even when ``retryWrites`` has been
+enabled on the ``MongoClient``. In addition, drivers MUST NOT add
```

Comment:
```suggestion
set to true on the ``MongoClient``. In addition, drivers MUST NOT add
```

Reply:
this sounds a little more clear to me but feel free to accept or reject, ditto below